datanode/gateway/graphql/schema.graphql

```diff
@@ -366,6 +366,14 @@ type Query {
   "Find a deposit using its ID"
   deposit("ID of the Deposit" id: ID!): Deposit
 
+  "Fetch all deposits"
+  deposits(
+    "Date range to fetch deposits between"
+    dateRange: DateRange,
+    "Pagination options"
+    pagination: Pagination,
+  ): DepositsConnection
```

Comment:
Shouldn't this take an optional party?

Reply:
No, we already have a depositsConnection subquery to return the deposits for a particular party defined in the `party` type. I added these deposits and withdrawals to allow the user to request list of all deposits/withdrawals as requested in the ticket, so that the GraphQL API was inline with REST API change requested.